### PR TITLE
HIVE-27286: Upgrade jettison version to 1.5.4 to address CVE-2023-1436

### PR DESCRIPTION
**Pull request by shreeyasand**
What changes were proposed in this pull request?
Upgrade jettison version to 1.5.4

Why are the changes needed?
To fix CVE-2023-1436 

Does this PR introduce any user-facing change?
No

How was this patch tested?
Existing tests

### CODE REVIEW

**Comment by akshat0395** [comment]
@shreeyasand jettison version 1.3.4 is a transitive dependency in tez-api as well, please exclude
Dependency tree:
|  +- org.apache.tez:tez-api:jar:0.10.2:compile
|  |  +- commons-lang:commons-lang:jar:2.6:compile
|  |  +- org.codehaus.jettison:jettison:jar:1.3.4:compile

**Comment by TuroczyX** [comment]
Nice catch @akshat0395! Otherwise, LGTM

File: standalone-metastore/metastore-tools/pom.xml [hunk 1]
```diff
@@ -81,6 +81,10 @@
             <groupId>org.apache.curator</groupId>
             <artifactId>curator-client</artifactId>
           </exclusion>
+          <exclusion>
```
Comment (akshat0395) [inline standalone-metastore/metastore-tools/pom.xml:84]: Jettison dependency that needed to be excluded is coming from tez-api as mentioned in my previous comment and not hive-common please exclude from there.
[depend.txt](https://github.com/apache/hive/files/11302939/depend.txt)

Please refer to the attached dependency tree to verify.

**Review by akshat0395 — APPROVED** [review]

**Comment by akshat0395** [comment]
LGTM, +1

**Comment by shreeyasand** [comment]
@TuroczyX could you please help merge this PR?

**Comment by shreeyasand** [comment]
@TuroczyX could you please help merge this PR?

**Comment by shreeyasand** [comment]
@veghlaci05 Could you please review and merge this?

**Comment by TuroczyX** [comment]
@deniskuzZ @veghlaci05 Could you please merge?

**Comment by shreeyasand** [comment]
@deniskuzZ @veghlaci05 could you please merge?

**Pull request merged**